src/commandline_frame.ts

```diff
@@ -250,7 +250,7 @@ export function refresh_completions(exstr) {
     if (!activeCompletions) enableCompletions()
     return Promise.all(
         activeCompletions.map(comp => comp.filter(exstr).then(resizeArea)),
-    )
+    ).catch(_ => undefined) // ignore errors
```

Comment:
shouldn't we console.log instead?

Reply:
It seems we often have completions failing so I thought it'd spam the user's console and they might not like it. But it's true that having a trace of things going wrong would be useful.